compiler/src/dotty/tools/dotc/transform/localopt/FormatChecker.scala

```diff
@@ -267,6 +276,8 @@ class TypedFormatChecker(partsElems: List[Tree], parts: List[String], args: List
         case Some(cc) => new Conversion(m, i, kindOf(cc(0))).tap(_.verify)
         case None     => new Conversion(m, i, ErrorXn).tap(_.errorAt(Spec)(s"Missing conversion operator in '${m.matched}'; $literalHelp"))
     end apply
+    // construct a default %s conversion
+    def apply(i: Int): Conversion = new Conversion(formatPattern.findAllMatchIn("%").next(), i, StringXn)
```

Comment:
Could you rename it to something more self-explanatory? e.g. `defaultConversion`